src/components/v5/common/ActionSidebar/partials/forms/CreateNewTeamForm/utils.tsx

```diff
@@ -1,20 +1,38 @@
+import { DecisionMethod } from '~types/actions.ts';
 import { type Colony } from '~types/graphql.ts';
+import { extractColonyRoles } from '~utils/colonyRoles.ts';
+import { extractColonyDomains } from '~utils/domains.ts';
 import { sanitizeHTML } from '~utils/strings.ts';
 
 import { type CreateNewTeamFormValues } from './consts.ts';
 
 export const getCreateNewTeamPayload = (
   colony: Colony,
   values: CreateNewTeamFormValues,
-) => ({
-  ...values,
-  domainName: values.teamName,
-  isCreateDomain: true,
-  colonyAddress: colony.colonyAddress,
-  colonyName: colony.name,
-  customActionTitle: values.title,
-  motionDomainId: values.createdIn,
-  annotationMessage: values.description
-    ? sanitizeHTML(values.description)
-    : undefined,
-});
+) => {
+  const baseDomainPayload = {
+    ...values,
+    domainName: values.teamName,
+    isCreateDomain: true,
+    colonyAddress: colony.colonyAddress,
+    colonyName: colony.name,
+    customActionTitle: values.title,
+    annotationMessage: values.description
+      ? sanitizeHTML(values.description)
+      : undefined,
+  };
+
+  if (values.decisionMethod === DecisionMethod.Permissions) {
+    return {
```

Comment:
we can just return the payload without the spread

Reply:
Ah yes you're right. This used to have other fields apart from the baseDomainPayload which I removed last minute